### ISSUE TITLE
Dual Monitors query

### ISSUE THREAD

**Issue by nomad101** [writemonkey/wm3#320]
Hello, thanks for producing such a great piece of software.

Apologies, if this is obvious. 
I plan to buy a second monitor. Is it possible to work on two different files within a project across the screens? For example, I could from one file (monitor 1), copy a section of text, and then paste into the another file (monitor 2). 

Thanks

**Comment by writemonkey** [comment]
No, it is not possible. Only one instance on wm at a time. What you can do is put a secondary text into repository and then choose edit to have that text in a split window. That doesn't help with 2 monitors, though.
i.
